mne/event.py

```diff
@@ -1499,3 +1499,37 @@ def match_event_names(event_names, keys, *, on_missing='raise'):
 
     matches = sorted(set(matches))  # deduplicate if necessary
     return matches
+
+
+def count_events(events, ids=None):
+    """Count events.
+
+    Parameters
+    ----------
+    events : ndarray, shape (N, 3)
+        The events array (consisting of N events).
+    ids : array-like of int | None
+        If None, count all event types present in the input. If array-like of
+        int, count only those event types given by ``ids``.
+
+    Returns
+    -------
+    counts : dict
+        A dictionary containing the event types as keys with their counts as
+        values.
+
+    Examples
+    --------
+        >>> events = np.array([[0, 0, 1], [0, 0, 1], [0, 0, 5]])
+        >>> count_events(events)
+        {1: 2, 5: 1}
+        >>> count_events(events, ids=[1, 5])
+        {1: 2, 5: 1}
+        >>> count_events(events, ids=[1, 11])
+        {1: 2, 11: 0}
```

Comment:
```suggestion
    >>> events = np.array([[0, 0, 1], [0, 0, 1], [0, 0, 5]])
    >>> count_events(events)
    {1: 2, 5: 1}
    >>> count_events(events, ids=[1, 5])
    {1: 2, 5: 1}
    >>> count_events(events, ids=[1, 11])
    {1: 2, 11: 0}
```

Reply:
I think indentation is necessary to render it as code (monospaced) in our docs:
https://output.circle-artifacts.com/output/job/412510ef-f25b-4a84-8ebe-a2eea11c9fda/artifacts/0/dev/generated/mne.count_events.html

Reply:
most of our doctest examples are indented, but @agramfort is right that it's not necessary. This one isn't indented in the source, and looks correct: https://output.circle-artifacts.com/output/job/412510ef-f25b-4a84-8ebe-a2eea11c9fda/artifacts/0/dev/generated/mne.epochs.equalize_epoch_counts.html#mne-epochs-equalize-epoch-counts

Since we're not consistent, I'll just merge as-is. Thanks @cbrnr